src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/Hosting/FramebufferHost.cs

```diff
@@ -41,9 +41,22 @@ public class FrameBufferHost : SkiaHost, ISkiaApplicationHost, IXamlRootHost, ID
 		/// <remarks>
 		/// Environment.CommandLine is used to fill LaunchEventArgs.Arguments.
 		/// </remarks>
-		public FrameBufferHost(Func<WUX.Application> appBuilder)
+		public FrameBufferHost(Func<WUX.Application> appBuilder) : this(appBuilder, null)
+		{
+		}
+
+		/// <summary>
+		/// Creates a host for a Uno Skia FrameBuffer application.
+		/// </summary>
+		/// <param name="appBuilder">App builder.</param>
+		/// <param name="displayScale">Optional display scale override.</param>
+		/// <remarks>
+		/// Environment.CommandLine is used to fill LaunchEventArgs.Arguments.
+		/// </remarks>
+		public FrameBufferHost(Func<WUX.Application> appBuilder, float? displayScale)
```

Comment:
This value should be read from the built version of the builder, not as a parameter.